beacon-chain/types/crystallized_state.go

```diff
@@ -325,8 +328,7 @@ func (c *CrystallizedState) NewDynastyRecalculations(seed [32]byte) (*Crystalliz
 		TotalDeposits:              c.data.TotalDeposits,
 		CrosslinkRecords:           c.data.CrosslinkRecords,
 		DynastyStart:               c.data.LastStateRecalc,
```

Comment:
Should we update this value too?

Reply:
It's updated. dynasty_start becomes last_state_recalc as defined in the spec here:

`Set last_dynasty_start = crystallized_state.last_state_recalc`